ci.ps1

```diff
@@ -6,17 +6,21 @@ param(
 $secret = $env:AUTOMATOR_USER_KEY_SECRET
 $key_id = $env:AUTOMATOR_USER_KEY_ID
 
-Write-Output $testSuiteId
-
 function Get-Auth-Token {
+    $query = @{"query" = "mutation { createSessionUsingUserKey(input: {userKeyId:""$key_id"" secret:""$secret""}) { token }}"} | ConvertTo-Json -Depth 9
```

Comment:
Will double double-quotes work here?

Reply:
It's different to how it was before

Reply:
It actually works. And it should be this way as the windows PowerShell evaluates "" as "